requirements.txt

```
@@ -3,7 +3,7 @@ aiohttp~=3.7.2
 aiodns~=2.0.0
 aiofiles==0.6.0
 backoff~=1.10.0
-Django~=2.2.17  # LTS version, switch only if we have a compelling reason to
+Django~=2.2.19  # LTS version, switch only if we have a compelling reason to
```

I believe the short term solution would be updating this comment to remember about FIPS patched django, as reminder for next time dependabot starts a PR
```suggestion
Django~=2.2.19  # LTS version, switch only if we have a compelling reason to (Update FIPS django)
```